docs/changelog.md

```diff
@@ -37,6 +37,68 @@ It doesn't include
 ## Unreleased
 
 
+### Highlights for users
+
+* Initial support for muted users. (PRs #4774, #4730, #4710, #4736, #4711,
+  #4699, #4678)
+* Added a setting to choose whether external links should open in the
+  embedded browser or the external browser. (PR #4679)
+* Added a server-compatibility notice for self-hosted orgs running Zulip
+  Server versions before 2.0.0. (PR #4750)
+* Stream settings screen now has a "subscribe" button. And, when
+  unsubscribed, useless toggle switches ("Pinned", "Muted", and
+  "Notifications") are hidden. (#4715)
+* Updated the primary app color to match the new logo. (PR #4544)
+* Fixed a bug on Android where the app sometimes opened to an unexpected
+  narrow when opening the app from the background. (#4758)
+* Fixed a glitch where the org URL input screen would flash by before the
+  authentication screen, on some app startups or when the user tapped a
+  logged-out account in the account picker. (#4604)
+* Fixed misconfigurations causing the Play Store to not offer the Android
+  app on devices that don't have cameras (#4722) or WiFi capability (PR
+  #4761).
+* Continued work toward enabling Android share-to-Zulip. (#4514)
+* Unclear, little-used "create group PM" option in the compose box removed.
+  (PR #4777)
+* Miscellaneous UI improvements. (#4734; PRs #4717, #4749, #4716, #4697,
+  #4686, #3517, #3395)
```

Comment:
#3517 and #3395 seem like bugfixes, not UI improvements, and thus probably deserve their own bullet point.

Reply:
Yeah, a bullet point about fixing the keypad and letter emoji (those two issues) would likely be a good one.

Otherwise, a catch-all list like this doesn't go in this section. Short of going and looking up the issues/PRs by number, it doesn't add anything beyond the "other fixes and improvements" line at the end. And on the App Store or Play Store they don't get linkified, so by the time someone's going to the trouble of pulling up our GitHub repo and looking up an issue, if they just want to browse through changes then they have plenty of other ways to do so (including the GitHub release notes, which contain the developer highlights too.)